README.md

```diff
@@ -23,11 +23,18 @@ So here I am with an up-to-date menu focusing on protecting the user from toxic
 
 Requirements:
 
+ * [Curl](#Curl)
  * [Git](#Git)
  * [CMake](#CMake)
 
 If you have both of the above requirements you can skip to [cloning the git repo and generating the project files](#cloning-and-generating-project-files).
 
+### Curl
+
+Curl is used to grab things like authorization tickets making pulling rockstar ids and much more possible. It is required to download as cpr, a library we use needs it, if you haven't used it before you will need to download and install it.
+
+[Download Link](https://curl.se/download.html)
```

Comment:
Your operating system needs to really be behind on updates for curl not be present, additionally YimMenu only targets Windows10 and higher operating systems.

Reply:
oh no i use a custom os of windows so that was probably why